backend/run.sh

```
@@ -0,0 +1,7 @@
+# backend/run.sh
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)